_qs/200-workspace.md

```diff
@@ -65,6 +65,7 @@ So click `Next`, which will bring us to a page asking us if we should import the
 
 ![Import archetype](/img/qs/git-import-6.png)
 
+<!-- I don't understand the next sentence. -->
```

Comment:
Quite right. Should I open an issue? I can't offer a suggestion as I have no idea what the sentence references and Peter saw fit to include it. 

Anyway, I will remove the comment.